src/memory_target.c

```diff
@@ -83,3 +83,14 @@ umf_result_t umfMemoryTargetGetCapacity(umf_memory_target_handle_t memoryTarget,
     assert(capacity);
     return memoryTarget->ops->get_capacity(memoryTarget->priv, capacity);
 }
+
+umf_result_t
+umfMemoryTargetGetBandwidth(umf_memory_target_handle_t srcMemoryTarget,
+                            umf_memory_target_handle_t dstMemoryTarget,
+                            size_t *bandwidth) {
+    assert(srcMemoryTarget);
```

Comment:
I see that this is not a common practice in this file but we should check for arguments using ifs and return UMF_RESULT_ERROR_INVALID_ARGUMENT when needed

Reply:
Done.